test/NodeWatchFileSystem.test.js

```diff
@@ -19,6 +19,35 @@ var fileSubdir = path.join(fixtures, "subdir", "watched-file.txt");
 
 describe("NodeWatchFileSystem", function() {
 	this.timeout(10000);
+
+	it('should throw if \'files\' argument is not an array', function() {
+		should(function() { new NodeWatchFileSystem().watch(undefined) }).throw(Error);
```

Comment:
Would it be possible to change these to have `.throw` assert the error message too?

```js
should(function() {
  new NodeWatchFileSystem().watch(undefined);
}).throw("Invalid arguments: 'files'");
```